setup.py

```diff
@@ -7,7 +7,7 @@
 from setuptools import setup, find_packages
 
 NAME = "vsts-cd-manager"
-VERSION = "0.118.0"
+VERSION = "1.0.0"
```

Comment:
It's up to your team, but on our team we use the sprint number as the minor version. That's how we got 118. It gives team members a good idea of when the features actually shipped without encoding a date.